src/ocaml/mode.mli

```diff
@@ -94,3 +98,49 @@ module Dict : sig
   end
 end
 with type mode := t
+
+(** [Select] is a utility module that represents a mode selection. *)
+module Select : sig
+  type mode = t
```

Comment:
@emillon can you remove this type alias?

Reply:
69a623c77e5a08bc7df69a26ed2b10c622f8c274